### ISSUE TITLE
Automatic theming of UPE

### ISSUE THREAD

**Issue by bborman22** [Automattic/woocommerce-payments#1827]
The goal of this issue is to theme the UPE element with a base set of styles pulled from the store's theme. The three primary pieces are: inputs and labels, fonts, and tabs. There are existing proofs of concept for each piece linked below. Styling of the element states (hover, focus, etc) will be handled in a separate issue. In this context tabs refers to the payment method selectors.

**Original UPE**
![Original UPE](https://user-images.githubusercontent.com/68524302/118538804-a929fa80-b71c-11eb-95e6-f371b58c53f7.png)

**Matching Storefront theme**
<img width="318" alt="storefrontcard" src="https://user-images.githubusercontent.com/68524302/118538818-acbd8180-b71c-11eb-81bc-3c80ed8bd9b0.png">

Inspiration for these updates can be drawn from the following proofs of concept:
[Inputs and labels](https://github.com/Automattic/woocommerce-payments/pull/1703)
[Tabs](https://github.com/Automattic/woocommerce-payments/pull/1736)
[Fonts](https://github.com/Automattic/woocommerce-payments/issues/1723)
[Discussion](paJDYF-1vP-p2)

**Comment by brezocordero** [comment]
Blocked by #1826 
The functions and tests are ready on this branch: https://github.com/Automattic/woocommerce-payments/tree/upe/1827-automatic-theming
The work can continue after the issue above is done.
